docs/index.rst

```diff
@@ -24,6 +24,7 @@ There are also many `downloadable examples documented in Sphinx
    Automated_wheels_building_with_scripts.rst
    Prerequisites.rst
    Detailed_build_instructions.rst
+   Quick_start_guide.rst
```

Comment:
+1

I guess there is a question wether this repository is mainly for api developers or should also serve as entry point for api users.